### DISCUSSION TITLE
AnduinOS new version release schedule

### DISCUSSION THREAD

**Discussion by Anduin2017** [Anduin2017/AnduinOS#34] [Announcements]
# AnduinOS Version Release schedule

AnduinOS Released | Release Date | Based On | End of Standard Support | End of Ubuntu Pro Support | End of Legacy Support
-- | -- | -- | -- | -- | --
1.2 | 2025-01-09 | Oracular Oriole | Jul 2025 |   |  
1.1 LTS | 2025-01-06 | Noble Numbat | Apr 2029 | Apr 2034 | Apr 2036
1.0 LTS | 2024-09-01 | Jammy Jellyfish | Apr 2027 | Apr 2032 | Apr 2034

Before AnduinOS was restructured, its Major Version was always 1.

Each time AnduinOS updates its corresponding Ubuntu code name, the Minor Version is incremented by 1. For example, `Noble` is `1.1`, and `Oracular` is `1.2`. In the future, AnduinOS will continue to align with new Ubuntu code names as they are introduced.

After AnduinOS is released, all package security updates and standard support will follow Canonical's support timeline for Ubuntu. Therefore, the corresponding LTS versions can receive up to five years of standard support.

Like Ubuntu, AnduinOS offers both LTS versions and short-term support versions. We recommend regular users download the latest LTS version.

Each Minor Version, such as 1.1, will continue to receive some bug fix updates. These updates will increase its Patch Version, for example, upgrading from 1.1.0 to 1.1.1. We always recommend users download the latest Patch Version.

For instance, AnduinOS might currently support the following versions:

- 1.2  
- 1.1 LTS  
- 1.0 LTS  

We recommend users download the latest Minor Version, which in this case is 1.1. Within 1.1, if there are multiple Patch Versions available, such as:

- 1.1.3  
- 1.1.2  
- 1.1.1  
- ...

We suggest users download the latest Patch Version, which is 1.1.3.

If you are an experienced developer or a tech enthusiast looking to experience the latest kernel and new features, we recommend downloading the latest Minor Version. However, these versions may undergo frequent changes and introduce new features regularly.

![image](https://github.com/user-attachments/assets/695cb61c-62db-4060-ac24-dfbc320aaa89)

**Comment by Airnan946** [comment]
That is great! I can't wait to try the new version!
